### PR TITLE
@mzikherman => Various Tickets / Fixes 

### PR DESCRIPTION
**Pull request by damassi**
Addresses: 

- [Avatar, name, and meta line should all link to artist's page](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&modal=detail&selectedIssue=DISCO-342)
- [Bullet separating the meta info from the follow button should be #666](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&modal=detail&selectedIssue=DISCO-343)
- [WFY (web) - Update wording for sub-header and description](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&modal=detail&selectedIssue=DISCO-336)
- [Artist page carousel is too tall](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&modal=detail&selectedIssue=DISCO-306)
- [Works for you spacing issues](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=57&modal=detail&selectedIssue=DISCO-346)

layout

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit bc1028b87a44bad489e768e8570ee9b34ea20e06

https://deploy-preview-1343--artsy-reaction.netlify.com

File: src/Styleguide/Components/__stories__/EntityHeader.story.tsx [hunk 1]
```diff
@@ -21,6 +23,16 @@ storiesOf("Styleguide/Components", module)
...
+            href={`http://www.artsy.net/${props.href}`}
+            FollowButton={
+              <Toggle initial={true}>
+                {({ on, toggle }) => (
+                  <span onClick={() => toggle()} style={{ cursor: "pointer" }}>
+                    <Sans size="2">{on ? "Unfollow" : "Follow"}</Sans>
+                  </span>
+                )}
+              </Toggle>
+            }
```
Comment (damassi) [inline src/Styleguide/Components/__stories__/EntityHeader.story.tsx]: Update storybooks `EntityHeader` example with follow / unfollow toggle

File: src/Apps/WorksForYou/MarketingHeader.tsx [hunk 2]
```diff
@@ -9,16 +9,17 @@ import {
...
+  color,
   Flex,
   Sans,
   Separator,
   Spacer,
-  themeProps,
 } from "@artsy/palette"
 
+// prettier-ignore
+const COLLECT_URL = `${sd.APP_URL}/collect?split_test[new_collect_page]=new&acquireable=true}`
```
Comment (mzikherman) [inline src/Apps/WorksForYou/MarketingHeader.tsx]: sweet

File: src/Styleguide/Components/EntityHeader.tsx [hunk 3]
```diff
@@ -11,44 +13,71 @@ interface EntityHeaderProps {
...
+                  // Capture click event so that interacting with Follow doesn't
+                  // trigger Container's link.
+                  event.stopPropagation()
+                }}
+              >
                 {FollowButton}
-              </>
-            )}
-          </Flex>
+              </span>
```
[the remark this answers is not in the document]
Comment (mzikherman) [reply src/Styleguide/Components/EntityHeader.tsx]: I think the follow button may already handle this kinda thing: https://github.com/artsy/reaction/blob/f4395f19482b7917aa50813da8efde6da290a6ac/src/Components/FollowButton/FollowArtistButton.tsx#L63

**Comment by mzikherman** [comment]
Looks great!

File: src/Styleguide/Components/EntityHeader.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (damassi) [reply src/Styleguide/Components/EntityHeader.tsx]: `event.stopPropagation()` serves a different function than `event.preventDefault()`. Propagation refers to events bubbling up through the tree (say from one node to the next) while `preventDefault` prevents the browser from performing the default action tied to the event.

**Pull request merged**